packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -11,49 +11,55 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
-
-:root {
+.spectrum-SearchWithin {
   --spectrum-searchwithin-width: 250px;
```

Comment:
this should be 192px (size-2400) to match the rest
https://github.com/adobe/react-spectrum/blob/main/packages/%40adobe/spectrum-css-temp/components/textfield/index.css#L39

Reply:
Changed to 240px after feedback from Spectrum Design